---
permalink: /
title: "Carl Mathias Kobel, MSc"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---


I'm a bioinformatician and a PhD candidate in the MEMO group at NMBU, Norway. My perspective is that microbiomes are largely undervalued and that we should better understand the minute interactions within them. I have a big data inspired approach, enjoy tinkering with hardware, and like to build parallelizable bioinformatics pipelines to gain insights into large multi-omic microbiome datasets.

Looking for a postdoc position in the Copenhagen area from ~april 2025.

## First-author publications

  - *PhD thesis*: Kobel, A Rumen-Centric View of the Cattle Holobiont, NMBU Library, 2025, [preview pending]
  - *Software paper*: Kobel et al., CompareM2 is a genomes-to-report pipeline for comparing microbial genomes, preprint/bioRxiv, 2024, [[pdf](https://www.biorxiv.org/content/10.1101/2024.07.12.603264v2.full.pdf) \| [doi](https://doi.org/10.1101/2024.07.12.603264 ) \| [docs](https://comparem2.readthedocs.io/) \| [repo](https://github.com/cmkobel/CompareM2) \| [bioconda](https://anaconda.org/bioconda/comparem2)]
  - *Holo-omics review*: Kobel et al., Integrating host and microbiome biology using holo-omics, RSC Molecular Omics, 2024, [[pdf](https://github.com/cmkobel/cmkobel.github.io/raw/master/files/Kobel%20et%20al.%202024%20-%20Integrating%20host%20and%20microbiome%20biology%20using%20holo-omics.pdf) \| [doi](https://doi.org/10.1039/D4MO00017J)]

## Activities

### Upcoming
  - [CEH](https://ceh.ku.dk/) travelling researcher presentation. "CompareM2 is a genomes-to-report pipeline". Copenhagen University, Denmark. Date: 11:00, Wednesday, 4th of December 2024.
  - PhD defense. "The Cattle Holobiont". Venue: Husdyrfagbygget, Norwegian University of Life Sciences, Ås, Norway. Date: 12:00, Friday, 17th of January 2025.

### Previous presentations
  - [AHC](https://www.appliedhologenomicsconference.eu/) (Applied Hologenomics Conference) 2024, Copenhagen, DK, Presentation "Holo-omic Workflow Utilizing Network Analysis Reveals Compositional Bistability Between Rumen Microbiomes" [[youtube](https://www.youtube.com/watch?v=gucJB-GG0WQ)]
  - [CGIF](https://www.congressgastrofunction.org/) (Congress on Gastrointestinal Function) 2024, Urbana-Champaign, USA, Presentation "Holo-omic network analysis reveals bistability in the rumen microbiome"
  - [AusME](https://www.ausme-microbes.org.au/) (Australian Microbial Ecology Conference) 2022, Melbourne Australia, poster and pitch "Holo-omics and host-microbiome interactions in the herbivore rumen ([#18](https://ausme-2022.p.asnevents.com.au/days/2022-11-07/abstract/86540))".
  - [ISME](https://www.isme-microbes.org/) (International Society of Microbial Ecology) 2022, Lausanne Switzerland, poster "Holo-omics and host-microbiome interactions in the herbivore rumen".
  

### Teaching
  - NMBU Bio326 "Genomsekvensering; verktøy og analyser" 2023-2024 \| Development of dry-lab course material and exercises, [[link](https://www.nmbu.no/en/course/bio326)]
  - Course in Microbiology at Aarhus University Hospital 2021 for MDs specializing in Microbiology \| Rapid long-read sequencing of clinical isolates.


### Miscellaneous

#### Conda recipe maintainer

Maintaining the recipes for the following Conda packages.

  - [bioconda::comparem2](https://anaconda.org/bioconda/comparem2) [![Conda Downloads](https://img.shields.io/conda/d/bioconda/comparem2)](https://bioconda.github.io/recipes/comparem2/README.html)
  - [bioconda::gapseq](https://anaconda.org/bioconda/gapseq) [![Conda Downloads](https://img.shields.io/conda/d/bioconda/gapseq)](https://bioconda.github.io/recipes/gapseq/README.html)
  - [conda-forge::r-chnosz](https://anaconda.org/conda-forge/r-chnosz) ![Conda Downloads](https://img.shields.io/conda/d/conda-forge/r-chnosz)
  - [bioconda::assemblycomparator2](https://anaconda.org/bioconda/assemblycomparator2) ![Conda Downloads](https://img.shields.io/conda/d/bioconda/assemblycomparator2) (archived)

#### Open source wizardry

Contribution to the upstream code base of:

  - [snakemake](https://snakemake.github.io/)

